### PR TITLE
Feature/wsdev 870 card placeholder merge

### PR DESCRIPTION
**Pull request by Singh-Ish**
Adding placeholder for people card and figure 
- written test for the same 

Jira Ticket No: WSDEV 870

### CODE REVIEW

**Review by marceloglacial — CHANGES_REQUESTED** [review]

**Comment by HanqingZhou** [comment]
Great job!

File: src/components/Card/components/FigureVariantCard.tsx [hunk 1]
```diff
@@ -20,7 +20,11 @@ const FigureVariantCard: React.FC<FigureVariantCardProps> = ({
         <LinkWrapper link={link}>
             <figure>
                 <figcaption dangerouslySetInnerHTML={{ __html: caption }} />
-                <p itemProp='description'>{description}</p>
+                {description ? (
```
[the remark this answers is not in the document]
Comment (Singh-Ish) [reply src/components/Card/components/FigureVariantCard.tsx:23]: Removed the description condition from the figure card variant

File: src/components/CardPeople/CardPeople.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import data from './CardData.json'
```
[the remark this answers is not in the document]
Comment (Singh-Ish) [reply src/components/CardPeople/CardPeople.stories.tsx:1]: sorry Marcelo i have changed the name of the source file to card people's data as it has data related to people card only

File: src/components/Card/Card.stories.tsx [hunk 3]
```diff
@@ -66,16 +66,6 @@ VideoVariant.args = {
     header: videoVariant.header,
 }
```
Comment (marceloglacial) [inline src/components/Card/Card.stories.tsx:67]: Please fix lint errors

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: src/components/Card/Card.stories.tsx (hunk 3 above)
Comment (marceloglacial) [reply src/components/Card/Card.stories.tsx:67]: Check the button CTA stories

File: src/components/Card/components/FigureVariantCard.tsx [hunk 4]
```diff
@@ -20,7 +20,11 @@ const FigureVariantCard: React.FC<FigureVariantCardProps> = ({
         <LinkWrapper link={link}>
```
[the remark this answers is not in the document]
Comment (Singh-Ish) [reply src/components/Card/components/FigureVariantCard.tsx]: Made the changes

File: src/components/CardPeople/CardPeople.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+import getVariation from '../../functions/getVariation'
+import CardPeople from './CardPeople'
+import React from 'react'
+import { Story } from '@storybook/react'
+
+const { variations } = data
+
+export default {
+    component: CardPeople,
+    title: `Components/Card People`,
```
[the remark this answers is not in the document]
Comment (Singh-Ish) [reply src/components/CardPeople/CardPeople.stories.tsx:11]: Moved the people card story under card

File: src/components/CardPeople/CardPeople.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+const Template: Story = (args: any) => <CardPeople {...args} />
+
+export const PeopleCard = Template.bind({})
+const peopleCard = getVariation('peopleVariant', variations)
+PeopleCard.args = {
```
[the remark this answers is not in the document]
Comment (Singh-Ish) [reply src/components/CardPeople/CardPeople.stories.tsx:18]: change the name to base story under people card

File: src/components/CardPeople/CardPeople.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    link: peopleCard.link,
+    image: peopleCard.image,
+    firstName: peopleCard.firstName,
+    lastName: peopleCard.lastName,
+    description: peopleCard.description,
+}
+
+export const PeopleCardWithNoImage = Template.bind({})
+const peopleCardNoImage = getVariation('peopleVariantWithoutImage', variations)
+PeopleCardWithNoImage.args = {
```
[the remark this answers is not in the document]
Comment (Singh-Ish) [reply src/components/CardPeople/CardPeople.stories.tsx:29]: change the name to "WithNoImage" story under people card

File: src/components/Card/Card.stories.tsx (hunk 3 above)
Comment (Singh-Ish) [reply src/components/Card/Card.stories.tsx:67]: fixed the lint error

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**